client/components/BikePopup.tsx

```diff
@@ -1,31 +1,38 @@
 import { StyleSheet, Alert, View, Text, Pressable, Button } from 'react-native';
 import { Callout } from 'react-native-maps';
 import { Entypo } from '@expo/vector-icons'; 
-import { Bike } from '../types/types';
+import { Bike, BikeProp } from '../types/types';
+import { distToBike } from '../services/distanceCalc';
 
-const BikePopup = (props: Bike) => {
+const BikePopup = (props: BikeProp) => {
+    const bike = props.bike;
+    const distance = distToBike(props.userLocation, bike.location)
     return (
         <Callout
             tooltip={true}
         >
             <View style={styles.popupContainer}>
                 <View style={styles.display}>
-                    <Text style={styles.header}>{props.name}</Text>
+                    <Text style={styles.header}>{bike.name}</Text>
                     <Text 
                         style={
-                                (props.status == 'available') 
-                                ? styles.available : styles.unavailable
+                            (bike.status == 'available') 
```

Comment:
would it be possible to use === here since both are strings?

Reply:
Yes I almost forgot we're in JavaScript (mostly)! I'll fix that in my next PR.